partcad/src/partcad/runtime_python_conda.py

```
@@ -52,69 +65,127 @@
+                        "--json",
+                        "-p",
+                        self.path,
+                        "pip",
+                    ],
+                    stdout=subprocess.PIPE,
+                    stderr=subprocess.PIPE,
+                    shell=False,
+                    encoding="utf-8",
+                )
+                _, stderr = p.communicate()
+                p.returncode
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)